eng/Analyzers.targets

```diff
@@ -13,5 +12,9 @@
     <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <AnalysisLevel>latest</AnalysisLevel>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <!--
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Comment:
What does this thing does? Do we still get all analyzers with this change?

Reply:
It enables all of the in-box analysers - I've commented it out for now as it adds _a lot_ of new errors (whether they're valid or not).

Reply:
OK, it's something we can address in follow-up. 

Btw, have you thought about enabling the new "artifacts" folder? Looks neat.

Reply:
Yep, I was just going to wait for your changes (and #1134) first.

I've been turning it on in various other repos of mine this morning after I had issues with it testing with preview 3.